### DISCUSSION TITLE
Is this just me?

### DISCUSSION THREAD

**Discussion by Two-Rivers-Obs** [Stellarium/stellarium#3413] [General]
Hey all, is anyone else having the issue of Stellarium crashing after about 2 minutes? Just since the last update, the software has been crashing every time I use it. Never had an issue before the update, but now it has become almost unusable. If there is a log file I can look at or upload to see what is causing the crash, where would I find it?

**Comment by ilkant** [comment]
Stellarium 23.2 User guide https://stellarium.org/files/guide.pdf chapter 5.3 The Logfile. It's in User directory.

If the Stellarium has not crashed, you can get log with F1 and there tab Log.
